### DISCUSSION TITLE
Ability to import all issues with given criteria

### DISCUSSION THREAD

**Discussion by coot** [community/community#7895] [Projects and Issues] [answered]
Often we have sophisticated labelling system, for various kinds of issues.  To migrate to the new project there should be a way to associate a bulk of issues satisfying some kind of search criteria with a new project.

**Comment by Flamenco** [comment]
A prime example of this would be to add all missing issues from 2 repositories that both have a common milestone. 

A workaround (or substitute) would be to batch add items from the issue list or milestone list to the project, but this is also not implemented.

**Comment by omerbensaadon** [comment, marked as answer]
We have some features in the new change log which helps to address this: 
https://github.blog/changelog/2022-04-07-the-new-github-issues-april-7th-update/

**Reply by omerbensaadon** [reply]
Namely, bulk adding to projects from Repo Issue list and the new Auto-Add Action

**Comment by coot** [comment]
@omerbensaadon thanks for letting us know; this seems what I was looking for, although it's still impossible to use since one has to have organisation level permissions (which are impossible to get in many enterprise environments).
